application.properties

```diff
@@ -1,8 +1,13 @@
 #Grails Metadata file
-#Thu Mar 07 09:37:04 EST 2013
-app.grails.version=1.3.7
+app.grails.version=2.5.6
 app.name=auv
 app.servlet.version=2.4
-app.version=0.2.9
-plugins.debug=1.0.2
-plugins.tomcat=1.3.7
+app.version=0.2.1
```

Comment:
how come you dropped the version number?

Reply:
copy and paste accident from ADV